src/taco-dependency-installer/elevatedInstallerProtocol.ts

```diff
@@ -23,6 +23,7 @@ module ElevatedInstallerProtocol {
         CompletedWithErrors = 12,
         NoAdminRights = 33,
         CouldNotConnect = 37,
+        RefusedPrompt = 42,
```

Comment:
Don't think there is a way to predict the future for that =/

I'll add links to the pages where I got my information from